components/webui/client/tsconfig/tsconfig.app.json

```
@@ -2,7 +2,6 @@
   "extends": "./tsconfig.base.json",
   "include": [
     "../src",
```

not sure how we missed this previously
```suggestion
    "../src/**/*",
```